aptos-move/vm-genesis/Cargo.toml

```diff
@@ -17,7 +17,7 @@ aptos-cached-packages =  { workspace = true }
 aptos-crypto = { workspace = true }
 aptos-framework =  { workspace = true }
 aptos-gas-schedule = { workspace = true } 
-aptos-types = { workspace = true }
+aptos-types = { workspace = true, features = ["testing"] }
```

Comment:
Same as above.

Reply:
This is needed, it's too cumbersome to extract a subset used in vm-genesis, and it's some lazy-parsed JSON.
So let's keep this as a visible wart.